iceoryx_posh/test/moduletests/test_popo_chunk_receiver.cpp

```diff
@@ -236,4 +237,51 @@ TEST_F(ChunkReceiver_test, Cleanup)
     EXPECT_THAT(m_memoryManager.getMemPoolInfo(0).m_usedChunks, Eq(0U));
 }
 
+TEST_F(ChunkReceiver_test, asStringLiteralConvertsChunkReceiveResultValuesToStrings)
+{
+    ::testing::Test::RecordProperty("TEST_ID", "5cbbda34-8a22-4eab-a8b6-20da345c1707");
+    using ChunkReceiveResult = iox::popo::ChunkReceiveResult;
+
+    // each bit corresponds to an enum value and must be set to true on test
```

Comment:
This whole test construct is pretty neat. We need some kind of tricks and hacks docu where this is mentioned as a template how one could test enum to string translation.

Reply:
Or even better ... (your are a victim of your own success!)

Can we create a macro a nifty macro where this is abstracted and one has to only write the enum value and the string translation?

Reply:
I have to think about this but it should be possible. With the macro it would even be possible to prevent one last case where one could add a value to the enum and forget to add it to the initializer list.